static/skywire-manager-src/src/app/services/vpn-auth-guard.service.ts

```diff
@@ -0,0 +1,44 @@
+import { Injectable } from '@angular/core';
+import { ActivatedRouteSnapshot, CanActivate, Router, RouterStateSnapshot, CanActivateChild } from '@angular/router';
+import { Observable, of } from 'rxjs';
+
+ /**
+ * Redirects the user to the error page of the VPN client if the lastError property is set. It
+ * must be used in the canActivate and canActivateChild properties of the routing module.
+ */
+@Injectable({
+  providedIn: 'root'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skywire/pullRequest?prid=7163595)
```suggestion
  providedIn: "root"
```